src/classes/UTIL_MasterSchedulableHelper.cls

```diff
@@ -111,7 +110,7 @@ public without sharing class UTIL_MasterSchedulableHelper {
                     jobsToAbort.add('NPSP 02 - Opportunity Contact Rollups (New)');
                     jobsToAbort.add('NPSP 03A - Opportunity Account Rollups LDV (New)');
                     jobsToAbort.add('NPSP 03B - Opportunity Contact Rollups LDV (New)');
-                    jobsToAbort.add('NPSP 04 - Opportunity Soft Credit Rollups (New)');
+                    jobsToAbort.add('NPSP 04 - Opportunity Soft Credit Rollups LDV (New)');
```

Comment:
Since this text is user-facing we shouldn't use the LDV acronym.

Reply:
@njjc  I was wondering about this, along with the staggered job times. 
* NPSP 01 - Opportunity Account Rollups (New) -- 11:00 PM
* NPSP 02 - Opportunity Contact Rollups (New) -- 11:10 PM
* NPSP 03A - Opportunity Account Rollups LDV (New) -- 11:20 PM
* NPSP 03B - Opportunity Contact Rollups LDV (New) -- 11:30 PM
* NPSP 04 - Opportunity Soft Credit Rollups LDV (New) -- 11:50 PM
* NPSP 05 - GAU Allocation Rollups LDV (New) -- 11:00 PM

My goal was keep the same naming pattern/sequence, but since one new batch job was needed it ended up as 3A and 3B. I'm definitely open to other naming patterns. Just not sure of another good term to use to describe the LDV type job and differentiate from the other jobs listed. The main difference is which batch job class gets executed. I suppose we could go with something like "Contact Rollups Job 1" and "Contact Rollups Job 2", though it's not very explanatory. 

I'm not tied to the `(New)` text either. I added that initially just to make it easier for me know when the jobs were scheduled. At this point I think we can drop `(New)`.

Reply:
I think including the word "Customizable" is a good idea, instead of (New). I'm fine with 3A and 3B so that we don't have to reconfigure everything. I don't have a strong preference on the timing; the only potential hangup I can think of is row locking errors if we're updating a contact and account at the same time, but I don't think we've hit that issue today and they're all kicked off at the same time.

Some day I'd love to come up with a better scheme so that we're not iterating over every record in the database every night... something like do the current full job weekly and the nightly jobs just iterate over Opportunity where LastModifiedDate is within the timeframe since the last batch job completed successfully... but that's a project for another time :)

Reply:
How about this?
* NPSP 01 - Customizable Rollups - Account Hard Credit
* NPSP 02 - Customizable Rollups - Contact Hard Credit
* NPSP 03A - Customizable Rollups - Account Hard Credit Job 2
* NPSP 03B - Customizable Rollups - Contact Hard Credit Job 2
* NPSP 04 - Customizable Rollups - Soft Credits
* NPSP 05 - Customizable Rollups - General Allocation Units

Reply:
How about "Contact/Account Hard Credit for Data Skew" or "for Large Data Volumes"? Curious what @smarovich thinks.

Reply:
While we're on this topic, should the "NPSP 06 - Recurring Donation Updates" job run at 11PM as well? That job can create/modify Opportunities - which are being rolled up by the various rollup jobs at the same time. Seems like that job should either start at 10PM or 12AM. I suppose that to handle a truly LDV org that has tens of millions of records, there may be a need to stagger that job much more than an hour or so.

Reply:
Here's what the jobs list looks like at the moment:

![image](https://user-images.githubusercontent.com/801017/34799463-e0bb62d0-f62d-11e7-9178-dacde4fbacdf.png)

Reply:
Good point, since it only creates open opps and we didn't used to roll those up it didn't matter. I think having the RD job first then the rollups jobs start an hour or two later sounds good.

Some LDV orgs have rollups jobs running longer than 24 hour s:(

Reply:
Hopefully our new engine will improve the job running times, but I still expect 30 million Opportunities to take a while. I'll move the RD start time to 10PM.

Reply:
@force2b I'm so sorry I missed this! I think the more info we can give in the name, the better. I like your proposed text "Contact Hard Credit for Large Data Volumes." That seems the most clear to me.